src/composables/config/useTemplateList.ts

```diff
@@ -114,13 +113,12 @@ export default () => {
         EmqxMessage.error(t('template.missingPluginInFile'))
       }
       if (name && plugin) {
+        isImportTemplate.value = true
         editTemplateData.value = jsonData
         templateDialogVisible.value = true
       }
     } catch (error) {
       EmqxMessage.error(t('common.jsonFormatError'))
-    } finally {
-      isImportTemplate.value = false
     }
```

Comment:
Why do we need to remove `isImportTemplate.value = false` if it has always been true?